corehq/messaging/smsbackends/twilio/models.py

```diff
@@ -54,12 +54,14 @@ def get_pass_through_opt_in_keywords(cls):
     def get_opt_out_keywords(cls):
         return ['STOP', 'STOPALL', 'UNSUBSCRIBE', 'CANCEL', 'END', 'QUIT']
 
-    def _convert_to_whatsapp(self, number):
-        assert WHATSAPP_PREFIX not in number, "Attempted to re-convert a number already formatted for Whatsapp"
+    @classmethod
+    def convert_to_whatsapp(cls, number):
+        if WHATSAPP_PREFIX in number:
```

Comment:
Could this say `number.startswith(WHATSAPP_PREFIX)`? Would be nice to be as restrictive as possible so as not to encourage a sloppy standard.

Reply:
sure, https://github.com/dimagi/commcare-hq/pull/27082